sockets/ssh_windows.go

```diff
@@ -0,0 +1,27 @@
+// +build windows
+
+package sockets
+
+import (
+	"errors"
+	"net"
+
+	"github.com/davidmz/go-pageant"
+	"golang.org/x/crypto/ssh"
+)
+
+func dialSSH(user, addr, socketPath string) (net.Conn, error) {
+	if !pageant.Available() {
+		return nil, errors.New("pageant not running")
+	}
+	auth := ssh.PublicKeysCallback(pageant.New().Signers)
+	sshClient, err := ssh.Dial("tcp", addr, &ssh.ClientConfig{
+		User:            user,
+		Auth:            []ssh.AuthMethod{auth},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
The [docs](https://github.com/golang/crypto/blob/master/ssh/client.go#L229..L231) say:
```
// InsecureIgnoreHostKey returns a function that can be used for
// ClientConfig.HostKeyCallback to accept any host key. It should
// not be used for production code.
```
I think that is sound advice.